### DISCUSSION TITLE
Platformio dim active regions

### DISCUSSION THREAD

**Discussion by oscarsan1** [arendst/Tasmota#14103] [Q&A] [answered]
Hi, 
Plaformio is dimming regions that are active. 
For exemple in xdrv_16_tuyamcu.ino the USE_LIGHT appears as not active but is defined in my_user_config.h: 

#define USE_LIGHT     

![imagen](https://user-images.githubusercontent.com/10104260/146671725-2caeee15-bc96-48b3-a13d-1ae48c0d8748.png)

is there any missing configuration?

**Comment by barbudor** [comment, marked as answer]
It's not platformio, it's Vscode C++ extension that can't deal with arduino project (there's generally a warning 1st time you open an ino file) 
Just disable dimming in C++ extensio settings
